src/cmd/isolated_server.cpp

```diff
@@ -128,25 +135,34 @@ int main(int argc, const char* argv[]) {
       LOG_GENERAL(WARNING, "AccountStore::RefreshDB failed");
     }
 
-    uint64_t blocknum;
+    mediator.RegisterColleagues(nullptr, &node, &lk, vd.get());
 
-    try {
-      blocknum = stoull(blocknum_str);
-    } catch (exception& e) {
-      cerr << "Error: "
-           << "blocknum not numeric" << endl;
-      return ERROR_IN_COMMAND_LINE;
-    }
+    AccountStore::GetInstance().InitSoft();
 
-    mediator.RegisterColleagues(nullptr, &node, &lk, vd.get());
+    uint64_t blocknum;
 
-    AccountStore::GetInstance().Init();
-    if (readAccountJsonFromFile(accountJsonFilePath)) {
-      cerr << "ERROR: "
-           << "Unable to parse account json file" << endl;
+    if (!loadPersistence && accountJsonFilePath.empty()) {
```

Comment:
What if I want to launch isolated server from clean slate?

Reply:
then just use the `-f` version